.github/workflows/ci.yml

```diff
@@ -239,4 +239,23 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
-      - run: DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker build -f papyrus_utilities.Dockerfile .
\ No newline at end of file
+      - run: DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 docker build -f papyrus_utilities.Dockerfile .
+    
+  random-table-test:
+    runs-on: ubuntu-latest
+    steps:
+      # run this job only if the path 'crates/papyrus_storage/src/db/**' is changed, because it takes around 2 minutes.
+      - uses: dorny/paths-filter@v3
+        id: changes
+        with:
+          filters: |
+            target_directory:
+              - 'crates/papyrus_storage/src/db/**'
```

Comment:
@DvirYo-starkware if you want some action to be executed only when a certain file/path changes, you can use the built-in syntax like in this example https://github.com/starkware-libs/papyrus/blob/2511b3a5a67ad6aef2b8c8aa35cd5a7a2de3d627/.github/workflows/helm-install.yml#L10